features/performance/hack/common.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+PERFORMANCE_DIR="$(
+    cd "$(dirname "$BASH_SOURCE[0]")/../"
+    echo "$(pwd)"
+)"
+
+ASSETS_DIR="${PERFORMANCE_DIR}/assets"
+MANIFESTS_DIR="${PERFORMANCE_DIR}/manifests"
+MANIFESTS_GENERATED_DIR="${MANIFESTS_DIR}/generated"
+TEMPLATES_DIR="${MANIFESTS_DIR}/templates"
+
+ISOLATED_CPUS=${ISOLATED_CPUS:-0}
```

Comment:
I wonder if we should provide a way to guesstimate the right value

Reply:
If you have the way to do it, I will happily add it

Reply:
Sure, and could be done as extension on top of this PR.
Unfortunately I don't have any clear path to this. The things we need to know to provide some helpful hints are:
- amount of cores in the target worker pool (assuming each worker has the same CPU model, otherwise is too complex)
- is HT enabled or not

I guess we can move the discussion in a another place.

Reply:
My immediate gripe is that the default config seems incorrect. I think we should bail out with error if ISOLATED_CPUS is equal to RESERVED_CPUS.

Reply:
will 0 and 1 work for you as initial values?

Reply:
I'd go with the second option: drop an error in the case ENV vars aren't defined.
If we can't give a reasonnable and working default config, we should give none.

Reply:
I just afraid that each time that you will run `make generate`, it will be very annoying to specify these variables